apps/src/sharedComponents/footer/CopyrightDialog.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import AccessibleDialog from '@cdo/apps/sharedComponents/AccessibleDialog';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import i18n from '@cdo/locale';
+
+import styles from '@cdo/apps/sharedComponents/footer/copyright-dialog.module.scss';
+
+export interface CopyrightDialogProps {
+  isOpen: boolean;
+  closeModal: () => void;
+}
+
+const CopyrightDialog: React.FC<CopyrightDialogProps> = ({
+  isOpen,
+  closeModal,
+}) => {
+  const onClose = () => {
+    closeModal();
+  };
+
+  const getDialogBody = () => {
+    return (
+      <div id="copyright-scroll-area">
+        <h3>{i18n.copyright()}</h3>
+        <hr />
+        <SafeMarkdown
+          markdown={i18n.copyright_thanks({
+            donors_link: pegasus('about/donors'),
+            partners_link: pegasus('about/partners'),
+            team_link: pegasus('about/team'),
+          })}
+        />
+        <p>{i18n.copyright_specialRecognition()}</p>
```

Comment:
Similarly to Stephen, have we looked at what this looks like with the `BodyThreeText`/`BodyFourText` or similar components? Does `SafeMarkdown` even support us using the typography components in its output?

Reply:
SafeMarkdown returns the components wrapped in a DIv: 
https://github.com/code-dot-org/code-dot-org/blob/5d7b3eed7c8bbf024d7764919002f65afb93ccf1/apps/src/templates/SafeMarkdown.jsx#L46-L58